packages/cli/internal/pkg/cli/context/manager.go

```diff
@@ -193,26 +193,6 @@ func (m *Manager) setOutputBucket() {
 	}
 }
 
-func (m *Manager) setTaskContext(contextName string) {
```

Comment:
This is a leftover from when we had two stacks for a context. It's not needed since `setContextEnv` is doing the real work